### PR TITLE
Feature/ua 763

### PR DESCRIPTION
**Pull request by vcward**
A shareable link is added to the single series and analyzer views for UHERO and NTA portals. Clicking the link icon automatically copies the URL to the user's clipboard.

Single Series:
Users should be able to share a selected date range

Analyzer:
Users should be able to share the series in the analyzer, the series drawn in the chart, a selected date range, tooltip options (name, units, geo), and visibility of YOY/YTD values.

The analyzer chart also now displays a legend at the bottom of the chart (also included in the export). All the changes should also be live on the staging server (http://uhero-bamboo.colo.hawaii.edu:3060/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Dataportal
 
-This project was generated with [angular-cli](https://github.com/angular/angular-cli) version 1.0.0-beta.30.
+This project was generated with [angular-cli](https://github.com/angular/angular-cli) version 1.4.8 (with NPM v. 3.10.10 and Node v. 6.10.3).
```
Comment (jonpage) [inline README.md]: We could also store this info in the package.json:
https://docs.npmjs.com/files/package.json#engines

**Comment by jonpage** [comment]
I agree with Dave's comments, except for the last one. I think leaving the shareable link in the location bar will add one more method for people to share the link. It also makes it more transparent how someone can get to a particular view in the analyzer. Anything we can do to make sharing easier will be worth it.

**Comment by vcward** [comment]
I added a title to the link similar to the Add/Remove to Analyzer button. It should display 'Copy Link' on hover then 'Link Copied' when clicking on it. It'll revert back to 'Copy Link' after 3 seconds. The link has also been moved to always show up to the right of the analyzer button.

File: README.md (hunk 1 above)
Comment (vcward) [reply README.md]: I think I'm going to do this in a separate branch. I tried updating Node, NPM, and Angular-cli and ran into some errors (not sure if there  are compatibility issues with Angular-cli and the latest Node LTS)

**Pull request merged**